### DISCUSSION TITLE
git clone is not working for a private repo

### DISCUSSION THREAD

**Discussion by BradThompsonEDCC** [community/community#23880] [New to GitHub]
<p>My college just created a new organization account. I created a new private repository. I got the clone string: <a href="https://github.com/MyCollege/BermudaTriange.git" rel="nofollow noopener">https://github.com/MyCollege/BermudaTriange.git</a></p>
<p>Then I created a directory and changed to it. Then I tried to use </p>
<p>C:\Tmp\test&gt;git clone <a href="https://github.com/MyCollege/BermudaTriangle.git" rel="nofollow noopener">https://github.com/MyCollege/BermudaTriangle.git</a><br>
Cloning into ‘BermudaTriangle’…<br>
remote: Repository not found.<br>
fatal: repository ‘<a href="https://github.com/MyCollege/BermudaTriangle.git/" rel="nofollow noopener">https://github.com/MyCollege/BermudaTriangle.git/</a>’ not found</p>
<p>I know the repo exists. I can use the Github desktop to make the clone onto my computer. I know that Git is working. I ran git clone on a personal repo and it worked.</p>
<p>What is the issue?</p>

**Comment by BradThompsonEDCC** [comment]
<p>I realize the initial post has “triangle” misspelled. However I’ve verified that the string given to me by github is the correct one.</p>
<p>git clone <a href="https://github.com/MyCollege/BermudaTriangle.git" rel="nofollow noopener">https://github.com/MyCollege/BermudaTriangle.git</a></p>
<p>This fails.</p>
<p>Using the Github desktop to clone the repo, it creates the repo just fine.</p>
<p>If I go into this repo and type git remote -v, I get the same string as above.</p>
<p>Going into a new directory and using git clone with it fails.</p>

**Comment by BradThompsonEDCC** [comment]
<p>git push is also not working:</p>
<p>remote: Repository not found.<br>
fatal: repository ‘<a href="https://github.com/MyCollege/BermudaTriangle.git/" rel="nofollow noopener">https://github.com/MyCollege/BermudaTriangle.git/</a>’ not found</p>
<p>Git desktop can push or pull changes. I cannot get the command line to work, even the dos window that Git Desktop supplied.</p>

**Comment by BradThompsonEDCC** [comment, marked as answer]
<p>It turned out that I had old session state:</p>
<ol>
<li>Open Control Panel from the Start menu</li>
<li>Select User Accounts</li>
<li>Select “Manage your credentials” in the left hand menu</li>
<li>Delete any credentials related to Git or GitHub</li>
</ol>
<p>Once I did this, it started working again.</p>

**Reply by niv3000** [reply]
thank you

**Comment by crbntyp** [comment]
<p>Perfect Thanks</p>

**Comment by MeghaNagari** [comment]
<p>Thanks,worked perfectly fine!! </p>

**Comment by liyi-1989** [comment]
<p>Thanks for your answer! Do you know is there any similar solution for Mac user? </p>

**Comment by nnimeshshah** [comment]
<p>Thank you, it worked for me too.</p>

**Comment by NadavAD** [comment]
<p>worked on mac too!</p>
<p>it’s because I have set up github before…</p>
<p>fo mac users see:</p>
<p><a href="https://uwaterloo.ca/information-systems-technology/services/nexus/nexus-troubleshooting-guide/clearing-cachedsaved-mac-os-x-credentials" rel="nofollow noopener">https://uwaterloo.ca/information-systems-technology/services/nexus/nexus-troubleshooting-guide/clearing-cachedsaved-mac-os-x-credentials</a></p>

**Comment by luthafiyyan** [comment]
<p>big thanks :+1:</p>

**Comment by HassanCheetah** [comment]
<p>Worked Thanks.</p>

**Comment by amulayprakash** [comment]
<p>Thanks due to your solution my things worked out </p>

**Comment by fillipemavasc** [comment]
<p>Excellent!!! I’ve searched a lot for a solution. And finally it worked :slight_smile:</p>
<p>Many thanks.</p>

**Comment by rajamedialabs** [comment]
<p>Thanks. Its Working.</p>

**Comment by sathishaworkwave** [comment]
<p>It’s working perfectly. Thank you.</p>

**Comment by Karthik0398** [comment]
<p>You just saved my pc from my anger !</p>

**Comment by trimaps** [comment]
<p>Thank you. worked for me. Magic!</p>

**Comment by bicotrucaltium** [comment]
<p>This is magic, Thank u a lot !!!</p>
